scripts/install-systemd-multi-user.sh

```diff
@@ -9,6 +9,38 @@ readonly SERVICE_DEST=/etc/systemd/system/nix-daemon.service
 readonly SOCKET_SRC=/lib/systemd/system/nix-daemon.socket
 readonly SOCKET_DEST=/etc/systemd/system/nix-daemon.socket
 
+
+# Path for the systemd override unit file to contain the proxy settings
+readonly SERVICE_OVERRIDE=${SERVICE_DEST}.d/override.conf
+
+create_systemd_override() {
+     echo "configure proxy settings for the nix-daemon service"
```

Comment:
Is this intended to be a `header` or a `subheader`?

Reply:
@grahamc I have changed `echo` to `header` and added Start and End messages to delimit the proxy configuration.  Sample log below.

~~~
---- Start: configure proxy for the nix-daemon service -------------------------
---- sudo execution ------------------------------------------------------------
I am executing:
    $ sudo mkdir -p /etc/systemd/system/nix-daemon.service.d

create directory for systemd unit override

---- sudo execution ------------------------------------------------------------
I am executing:

    $ sudo tee /etc/systemd/system/nix-daemon.service.d/override.conf

create systemd unit override
[Service]
Environment=http_proxy=http://webproxy.work.com:80
Environment=https_proxy=http://webproxy.work.com:80
Environment=ftp_proxy=http://webproxy.work.com:80
Environment=no_proxy=work.com,localhost,127.0.0.1
---- End: configure proxy for the nix-daemon service ---------------------------
~~~